test/parallel/test-whatwg-url-custom-parsing.js

```diff
@@ -55,11 +55,9 @@ for (const test of failureTests) {
     () => new URL(test.input, test.base),
     (error) => {
       assert.throws(() => { throw error; }, expectedError);
-
-      // The input could be processed, so we don't do strict matching here
-      let match;
-      assert(match = (`${error}`).match(/Invalid URL: (.*)$/));
-      assert.strictEqual(error.input, match[1]);
+      assert.strictEqual(`${error}`, 'TypeError [ERR_INVALID_URL]: Invalid URL');
+      assert.strictEqual(error.message, 'Invalid URL');
+      assert.strictEqual(error.input, test.input);
```

Comment:
```suggestion
      assert.strictEqual(`${error}`, 'TypeError [ERR_INVALID_URL]: Invalid URL');
      assert.strictEqual(error.message, 'Invalid URL');
```
This fails as the null character is omitted from the input as the removed comment says:
> The input could be processed, so we don't do strict matching here